docs/copied-from-beats/docs/loggingconfig.asciidoc

```diff
@@ -235,6 +235,12 @@ true.
 
 When true, logs messages in JSON format. The default is false.
 
+[float]
+==== `logging.ecs`
+
+When true, logs messages with minimal required Elastic Common Schema (ECS)
+information.
```

Comment:
This might be a good opportunity to link to the [ECS docs](https://www.elastic.co/guide/en/ecs/current/ecs-log.html). Something like this?

```suggestion
When true, logs messages with the minimum required {ecs-ref}/ecs-log.html[Elastic Common Schema (ECS)]
information.
```